qa/supply-chain/imports.lock

```diff
@@ -1,13 +1,62 @@
 
 # cargo-vet imports lock
 
+[[publisher.bridgetree]]
+version = "0.4.0"
+when = "2023-09-08"
+user-id = 6289
+user-login = "str4d"
+
 [[publisher.bumpalo]]
 version = "3.14.0"
 when = "2023-09-14"
 user-id = 696
 user-login = "fitzgen"
 user-name = "Nick Fitzgerald"
 
+[[publisher.equihash]]
+version = "0.2.0"
+when = "2022-06-24"
+user-id = 6289
+user-login = "str4d"
+
+[[publisher.f4jumble]]
+version = "0.1.0"
+when = "2022-05-10"
+user-id = 6289
+user-login = "str4d"
+
+[[publisher.halo2_gadgets]]
+version = "0.3.0"
+when = "2023-03-22"
+user-id = 1244
+user-login = "ebfull"
+
+[[publisher.halo2_legacy_pdqsort]]
+version = "0.1.0"
+when = "2023-03-10"
+user-id = 199950
+user-login = "daira"
+user-name = "Daira Hopwood"
```

Comment:
I wonder where it's getting this username. My https://crates.io/settings/profile says "Daira Emma Hopwood".
![image](https://github.com/zcash/zcash/assets/643204/4307e853-17dc-4d8c-aff0-17dec564804a)